commit 28dd6726e7bb85add8a8fe61438ed2d483acc1f7
Author: Rasmus Villemoes <rasmus.villemoes@prevas.dk>
Date:   Tue Feb 20 16:09:12 2018 +0100

    EDAC, layerscape: Allow building for LS1021A
    
    The LS1021A has a memory controller supported by this driver. It builds
    just fine, and I've done some rudimentary testing using the error
    injection facility, which suggests that it is indeed working.
    
    Signed-off-by: Rasmus Villemoes <rasmus.villemoes@prevas.dk>
    Acked-by: York Sun <york.sun@nxp.com>
    Cc: Alexander Stein <alexander.stein@systec-electronic.com>
    Cc: linux-edac <linux-edac@vger.kernel.org>
    Link: http://lkml.kernel.org/r/20180220150912.2954-1-rasmus.villemoes@prevas.dk
    Signed-off-by: Borislav Petkov <bp@suse.de>

diff --git a/drivers/edac/Kconfig b/drivers/edac/Kconfig
index 3c4017007647..cb4ff1cc6eb4 100644
--- a/drivers/edac/Kconfig
+++ b/drivers/edac/Kconfig
@@ -254,7 +254,7 @@ config EDAC_MPC85XX
 
 config EDAC_LAYERSCAPE
 	tristate "Freescale Layerscape DDR"
-	depends on ARCH_LAYERSCAPE
+	depends on ARCH_LAYERSCAPE || SOC_LS1021A
 	help
 	  Support for error detection and correction on Freescale memory
 	  controllers on Layerscape SoCs.